### PR TITLE
Internal: Use wp-lite-env in Playwright tests [EDS-650]

### PR DESCRIPTION
**Pull request by yotamselementor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
...
 		"setup-templates": "rm -rf ./templates && mkdir -p ./templates/playwright && mkdir -p ./templates/lighthouse && cp ./tests/playwright/templates/* ./templates/playwright && cp ./tests/lighthouse/templates/* ./templates/lighthouse",
-		"start-local-server": "npm run setup-templates && wp-env start",
+		"start-local-server": "npm run setup-templates && wp-lite-env start config=./tests/playwright/.playwright-wp-lite-env.json -- port=8888 && wp-lite-env start config=./tests/playwright/.playwright-wp-lite-env.json -- port=8889",
 		"test:php": "docker-compose run --rm wordpress_phpunit phpunit",
 		"test:php_multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit",
-		"test:setup:playwright": "wp-env run cli bash elementor-config/setup.sh && wp-env run tests-cli bash elementor-config/setup.sh",
+		"test:setup:playwright": "wp-lite-env cli config=./tests/playwright/.playwright-wp-lite-env.json -- port=8888 command=\"bash elementor-config/setup.sh\" && wp-lite-env cli config=./tests/playwright/.playwright-wp-lite-env.json -- port=8889 command=\"bash elementor-config/setup.sh\"",
```
Comment (blackcolt) [inline package.json:20]: is this the setup only for CI? or is it for local dev and CI?

File: tests/playwright/assets/wp-env-cli.ts [hunk 2]
```diff
@@ -1,25 +1,6 @@
...
-		github.warning( logs );
-	}
-
-	if ( logs.includes( 'Error' ) ) {
-		github.error( logs );
-	}
-
-	return logs;
+	const { cli } = await import( '@elementor/wp-lite-env' );
+	await cli( port, command );
```
Comment (blackcolt) [inline tests/playwright/assets/wp-env-cli.ts:5]: is this function in this PR or is it from somewhere else?

File: package.json (hunk 1 above)
Comment (yotamselementor) [reply package.json:20]: Not sure what you mean, it should work both locally and in the CI

File: tests/playwright/assets/wp-env-cli.ts (hunk 2 above)
Comment (yotamselementor) [reply tests/playwright/assets/wp-env-cli.ts:5]: I don't understand the question

**Review by blackcolt — DISMISSED** [review]

File: tests/playwright/assets/wp-env-cli.ts
Comment (davseve) [inline tests/playwright/assets/wp-env-cli.ts:1]: Maybe change the file name to `wp-lite-env-cli.ts`?
Though we will need to change all the `wpEnvCli` appearances to `wpLiteEnvCli` as well.

File: tests/wp-env/config/setup.sh [hunk 3]
```diff
@@ -25,5 +21,13 @@ for id in $(wp user list --field=ID)
 do wp user meta add "$id" "announcements_user_counter" 999
 done
 
+wp --user=admin elementor library import-dir /var/www/html/elementor-templates
+
+wp cache flush
+wp rewrite flush --hard
+wp elementor flush-css
+
+wp post list --post_type=page
```
Comment (davseve) [inline tests/wp-env/config/setup.sh:30]: I know that this function appears in the original code.
But do we really need it?

Comment (yotamselementor) [reply tests/wp-env/config/setup.sh]: We do not!
Done: https://github.com/elementor/elementor/pull/29345/commits/e67b47b51c5dc0d041bb3bb2418891596c005b3d

File: tests/playwright/assets/wp-env-cli.ts
Comment (yotamselementor) [reply tests/playwright/assets/wp-env-cli.ts:1]: Maybe we should rename it to `wp-cli.ts` and abstract the implementation out of the name? WDYT?

File: tests/playwright/upgrade-test/.upgrade-test-wp-lite-env.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+	"core": "6.7",
```
[the remark this answers is not in the document]
Comment (yotamselementor) [reply tests/playwright/upgrade-test/.upgrade-test-wp-lite-env.json:2]: Agreed, but I don't know what to do about that right now TBH

File: tests/playwright/assets/wp-env-cli.ts
Comment (davseve) [reply tests/playwright/assets/wp-env-cli.ts:1]: Great suggestion!

Comment (yotamselementor) [reply tests/playwright/assets/wp-env-cli.ts:1]: Fixed: https://github.com/elementor/elementor/pull/29345/commits/2c8a817e95ae431d2d93a9432960d256097ceb30

**Review by davseve — APPROVED** [review]

**Pull request merged**